byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +60,33 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
```

Comment:
This servlet focuses on getting the placeTypes that exist within the API and formatting them, such as `Convenience store` or `Park`. It was just a small detail since it loaded them as `convenience-store` and `park`. This function isn't used for loading the recommendations list, only the places to make them as interests for the buttons. There was one exception that I found, `Rv park` which should need the v capitalized, but I was going to filter through the interests at a later date to remove other placeTypes so I'll focus on that in a future PR

Reply:
Oh I see, my bad.